fingertips_py/api_calls.py

```diff
@@ -63,20 +63,39 @@ def get_json_return_df(url, transpose=True):
 
 def get_data_in_tuple(url):
     """
-
     :param url: A url to make a request
-    :return: A tuple of returned data
+    :return: A list of returned data in tuples
     """
     try:
         req = requests.get(url)
     except requests.exceptions.SSLError:
         req = requests.get(url, verify=False)
     json_resp = json.loads(req.content.decode('utf-8'))
-    tup = [tuple(d.values()) for d in json_resp]
-    if isinstance(tup[0][0], str):
-        return [(t[1], t[0]) for t in tup]
+    tup_list = []
+    for item in json_resp:
+        tup_list.append([(k, v) for k, v in item.items()])
+    if isinstance(tup_list[0][0], str):
+        return [(t[1], t[0]) for t in tup_list]
+    else:
+        return tup_list
+    
+    
+def get_data_in_dict(url, key = None, value = None):
```

Comment:
Is there any point of key being a parameter here? Won't any value assigned to it just be overwritten?